tensorflow_addons/utils/__init__.py

```diff
@@ -0,0 +1 @@
+from . import types

```

Comment:
nit: is it necessary?

Reply:
Can you explain your question? I don't really understand what you mean.

Reply:
I mean we can keep the file empty

Reply:
or set the `__all__` variable if necessary

Reply:
ho I see what you mean. Let me change that.